spec/src/main/asciidoc/core/injectionandresolution.asciidoc

```diff
@@ -24,14 +24,10 @@ The container is not required to support circular chains of dependencies where e
 
 Beans and their clients may be deployed in _modules_ in a module architecture.
```

Comment:
The way I understand it, the terms "modules" and "module architecture" are used here in a very abstract meaning. I agree it would be nice to change them, because JPMS exists today and people may easily mistake this "module architecture" for JPMS (which it is not!).

That said, I'd do it in a separate PR.